_config.yml

```diff
@@ -37,11 +37,17 @@ markdown:    kramdown
 lsi:         false
 excerpt_separator: "\n\n"
 
-# Serving upm.mraa.io
+# Serving from https://upm.mraa.io
+#detach:  false
+#port:    443
+#host:    upm.mraa.io
+#baseurl: "" # does not include hostname
+
+# Serving from https://intel-iot-devkit.github.io
```

Comment:
This is deploying to https://intel-iot-devkit.github.io/upm/ with success, but upm.mraa.io is failing